Changelog.md

```diff
@@ -7,6 +7,7 @@
 -   Support for Serial Terminal app.
 -   Warn about outdated app versions.
 -   Hotkey Ctrl-W or Cmd-W closes current window.
+-   Hotkey Ctrl-R or Cmd-R restarts current window instead of reloading.
```

Comment:
Do users understand, what the difference between reloading and restarting a window is or this just that it now works the way users expect it anyhow?

Reply:
Good question, and, I can't really know what users expect here. It's a desktop app, not a browser, but there is a browser in the app. Most desktop apps don't have a reload function, so most users might not expect one to exist. 

Restarting the app instead of reloading will give a more similar experience to electron 13 without process reuse, since that essentially started a new process as well. 

I guess I can rename the menu item to something like "Restart" or "Reload Window" (the latter inspired by vs code command palette).

Reply:
Yes, for users “Reload” really makes little sense. To them the app is restarted, so let us also use that term everywhere where it makes sense.

And, users will now observe any change of behaviour to “Ctrl-R”. So let us remove that from the changelog.